addons/website_blog/views/website_blog_templates.xml

```diff
@@ -93,7 +93,10 @@ list of filtered posts (by date or tag).
 <!-- (Option) Blog: Show latest-post as top banner
     Replace top-banner content with the latest published post
 -->
-<template id="opt_blog_cover_post" name="Top banner - Name / Latest Post" inherit_id="website_blog.blog_post_short" active="True">
+<!-- We set a priority to be sure the view is render after oe_structure
+     customization. This is needed if a block has been drop in the
+     `#oe_structure_blog_all_header.oe_structure`. -->
```

Comment:
@qsm-odoo is that ok for you ?

Reply:
@Guillaume-gdi Better but typos :grimacing: 

- "is render" is not english, while we are at it let's not use the term "render", let's go with something like `A priority is set so that this is applied after oe_structure customizations`.
- "has been drop" is not english either